relay-server/src/actors/project_buffer.rs

```diff
@@ -290,55 +338,26 @@ impl BufferService {
                     return Err(BufferError::Full(estimated_db_size));
```

Comment:
As a follow-up, we should: 
- try to spool as much as possible to fill up the disk (spool as many envelopes as possible)
- potentially: keep the rest in memory, and let the upstream service handle the case that we can no longer buffer. this must not "block" the caller on pending spooling, though.
- propagate an error up, via healthcheck and as 503 on queueing in the common endpoint handler.

Reply:
As agreed, adding this to followups and will jump one some of the items after merging this PR.